lib/matplotlib/backends/backend_wx.py

```diff
@@ -96,72 +72,52 @@ def DEBUG_MSG(string, lvl=3, o=None):
         #print >>sys.stderr, "%s- %s in %s" % (_DEBUG_lvls[lvl], string, cls)
         print("%s- %s in %s" % (_DEBUG_lvls[lvl], string, cls))
 
+
 def debug_on_error(type, value, tb):
     """Code due to Thomas Heller - published in Python Cookbook (O'Reilley)"""
     traceback.print_exc(type, value, tb)
     print()
     pdb.pm()  # jdh uncomment
 
+
 class fake_stderr(object):
-    """Wx does strange things with stderr, as it makes the assumption that there
-    is probably no console. This redirects stderr to the console, since we know
-    that there is one!"""
+    """
+    Wx does strange things with stderr, as it makes the assumption that
+    there is probably no console. This redirects stderr to the console, since
+    we know that there is one!
+    """
+
     def write(self, msg):
         print("Stderr: %s\n\r" % msg)
 
 #if _DEBUG < 5:
-#    sys.excepthook = debug_on_error
-#    WxLogger =wx.LogStderr()
-#    sys.stderr = fake_stderr
-
-# Event binding code changed after version 2.5
-if wx.VERSION_STRING >= '2.5':
-    def bind(actor,event,action,**kw):
-        actor.Bind(event,action,**kw)
```

Comment:
Based on this comment in code being taken out, it looks like Bind() became available after v2.5, so we should be good with getting rid of the old style.